packages/react-dev-utils/copyPublicFolder.js

```diff
@@ -0,0 +1,7 @@
+var fs = require('fs-extra');
```

Comment:
Still need to leave this function in `build.js` inline because I think people will want to change it.